code/modules/admin/admin.dm

```diff
@@ -68,7 +68,12 @@ GLOBAL_VAR_INIT(nologevent, 0)
 	var/body = "<html><head><title>Options for [M.key]</title></head>"
 	body += "<body>Options panel for <b>[M]</b>"
 	if(M.client)
-		body += " played by <b>[M.client]</b> "
+
+		if(check_watchlist(M.ckey)
+			body += " played by <a style = 'color: #db9737;' href = '?_src_=holder;watchsearch=[M.ckey];'><b>[M.client]</b></a> "
```

Comment:
spacing in html spans and what not is usually limited:
So instead of `<span style = 'color: #val' href = '?link'>` you can use `<span style='color: #value' href='?link'>`